.github/workflows/ci.yml

```
@@ -273,6 +273,19 @@ jobs:
           args: --package dotenv-linter
         if: steps.cache-target.outputs.cache-hit != 'true'
 
+  windows_install:
+    name: Install / Windows
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup dotenv-linter
+        shell: bash # it uses Git Bash, i.e. MinGW
+        run: |
+          mkdir -p $HOME/bin
+          sh "${GITHUB_WORKSPACE}/install.sh" -b $HOME/bin
+      - name: Verify successful installation
+        run: $HOME/bin/dotenv-linter -v
```

```suggestion
        run: $HOME/bin/dotenv-linter.exe -v
```
I suppose you forgot the `.exe` extension 🤔

---

Yeah, thanks :)